src/lib/components/Menu/Menu.svelte

```diff
@@ -45,33 +47,25 @@
 		setOrigin();
 	}
 
-
-	/**
-	 * Searches for the first parent node that can scroll
-	 * @see {@link https://thewebdev.info/2021/06/27/how-to-find-the-first-scrollable-parent-element-with-javascript/|This article by The Web Dev}
-	 */
+	// Searches for the first parent node that can scroll
+	// Source: https://thewebdev.info/2021/06/27/how-to-find-the-first-scrollable-parent-element-with-javascript/
```

Comment:
I'd suggest preferring JSDoc syntax, as it provides the info on hover, which greatly improves the DX. Though it's not uniform with the rest of the codebase, it would save a big touch if we decide to autogenerate documentation further down the line. Double slashes obfuscates code documentation.

Reply:
Hmm, I've heard of JSDoc but haven't used it personally. Can you provide an example of what you mean by "oh hover?". Does that require an editor plugin, I don't recall seeing that.

At least in VS Code you can control + click to open links in the current format.

I'd be interested in reviewing this. If we do it, I'd rather go all in rather than mixing here and there.